src/main/java/com/itspace/job/model/BlogCategory.java

```diff
@@ -13,13 +10,14 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @Builder
+@ToString(exclude = "")
 public class BlogCategory {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
     private String name;
-    @ManyToMany(mappedBy = "blogCategories")
+    @ManyToMany(mappedBy = "blogCategories",cascade={CascadeType.ALL})
     private List<Blog> blogs;
 
```

Comment:
arajarkum em @ToString(exclude = "") chtoxnel datark ev nshel filderi anunner@ 
orinak 
@ToString(exclude = "blogs")
http://www.javabyexamples.com/delombok-tostring/

Reply:
Thank you. I will  add the name of variables .